tests/run-macros/i8007/Test_4.scala

```diff
@@ -34,6 +39,11 @@ enum Opt[+T] {
   println(t5) // true
   println
 
+  // Here invariant case without explicit type parameter will instantiate T to OptInv[Any]
+  val t5_2 = OptInv.Sm[Int](23) === OptInv.Sm(23)
+  println(t5) // true
```

Comment:
if we keep this case, I made a mistake here this should be 
```suggestion
  println(t5_2) // true
```

Reply:
Thanks! Can you add a commit with both suggested changes to Test_4?

Reply:
Never mind, I need to change a comment anyway and can roll in the test.